decidim-core/spec/mailers/reported_mailer_spec.rb

```diff
@@ -119,7 +119,7 @@ module Decidim
             reportable.coauthorships.destroy_all
             create :coauthorship, coauthorable: reportable, author: meeting
 
-            expect(email_body(mail)).to match(translated(meeting.title))
+            expect(email_body(mail)).to include(decidim_escape_translated(meeting.title))
```

Comment:
This change isn't in develop. If we want to change it in v0.27 we should do it there first and then backport it

Reply:
reverted in [bde61ef](https://github.com/decidim/decidim/pull/12547/commits/bde61efea5e7275a335ef994faa30ba21f9aff15)